pkg/prometheus/store.go

```diff
@@ -70,9 +70,9 @@ func AddRemoteReadsToStore(ctx context.Context, store *assets.Store, namespace s
 	return nil
 }
 
-func AddAlertmanagerEndpointsToStore(ctx context.Context, store *assets.Store, namespace string, ams []monv1.AlertmanagerEndpoints) error {
+func AddAlertmanagerEndpointsToStore(ctx context.Context, store *assets.Store, namespace string, ams []monv1.AlertmanagerEndpoints, p monv1.PrometheusInterface) error {
```

Comment:
same remark here, there's no need for this function to exist in pkg/prometheus

Reply:
Where should we put it? there is no pkg/prometheus/server/store.go. Should it be in  pkg/prometheus/server/operator.go again?

Reply:
I don't see a problem in creating a `store.go` file 🤔

Reply:
Ok then, I will create a `store.go` file and add this function there.